### ISSUE TITLE
EPIC - Migrate kie-samples and serverless-logic-web-tools-accelerator-quarkus to kie-tools and use its repo as a deployment repo

### ISSUE THREAD

**Issue by fantonangeli** [apache/incubator-kie-issues#1638]
Migrate the [kie-samples](https://github.com/kiegroup/kie-samples) (to be renamed to `serverless-logic-web-tools-samples`) and the [SLWT Quarkus Accelerator](https://github.com/kiegroup/serverless-logic-sandbox-deployment/tree/quarkus-accelerator) (to be renamed `serverless-logic-web-tools-accelerator-quarkus`) to `kie-tools` repository. 
This task will align with the effort implemented in https://github.com/apache/incubator-kie-issues/issues/1090, using local bare Git repositories for development.
During this migration, the related repositories need to be created under Apache.

Tasks:
- [x] #1639
- [x] #1640

**Comment by tiagobento** [comment]
I don't think we need an EPIC of EPICs, so with the goal of making things simpler, I'm closing this one 😉

**Comment by fantonangeli** [comment]
Fine for me
